go.mod

```diff
@@ -13,9 +13,9 @@ require (
 	github.com/google/gops v0.3.18
 	github.com/gorilla/websocket v1.5.0
 	github.com/mitchellh/mapstructure v1.5.0
-	github.com/multiversx/mx-chain-core-go v1.2.1-0.20230322093158-35195fa155c0
+	github.com/multiversx/mx-chain-core-go v1.2.1-0.20230329082847-b78e96c3ad5a
```

Comment:
This is not the latest commit hash
I think this should be `932a718276f65ff8f820720adf5e98c4cfdd3074`.

Check here: https://github.com/multiversx/mx-chain-core-go/pull/171

Reply:
fixed